docs/build/basics/transactions.md

```diff
@@ -19,7 +22,7 @@ In order for a transaction to be valid and executed it must contain signatures f
 
 ![Screenshot 2023-08-17 at 14.52.56.png](_transactions_images/Screenshot_2023-08-17_at_14.52.56.png)
 
-**Script**
+#### Script
```

Comment:
Should this be ##/H2?  Why jump all the way down to ####?

Reply:
bumped it to h3